fendermint/app/options/src/lib.rs

```diff
@@ -76,15 +76,15 @@ pub struct Options {
     pub home_dir: PathBuf,
 
     /// Set a custom directory for ipc log files.
-    #[arg(short = 'l', long, env = "FM_LOG_DIR")]
+    #[arg(short = 'd', long, env = "FM_LOG_DIR")]
```

Comment:
This is already used for `home_dir`

Reply:
oh wow, I am blind I missed it completely :D